tests/Agent/MultiverseTesting/ConsoleScanner/Program.cs

```diff
@@ -75,6 +75,13 @@ public static void ProcessAssemblies(ScannerConfiguration configuration)
                 {
                     foreach (var intrumentedDllFileLocation in downloadedNugetInfo.InstrumentedDllFileLocations)
                     {
+                        // skip unusable frameworks like silverlight
+                        var targetFramework = Path.GetFileName(Path.GetDirectoryName(intrumentedDllFileLocation));
+                        if (!targetFramework.StartsWith("net") || targetFramework.StartsWith("net35"))
```

Comment:
Please review https://learn.microsoft.com/en-us/nuget/reference/target-frameworks#supported-frameworks and add ignores for any other undesirable target frameworks. 

Here are a few that I see:
- net11
- net20
- net40
- net403
- net4** up to < 4.6.2?
- netmf ? (.NET Microframework?)
- netcoreapp1.0
- netcoreapp1.1
- netcoreapp2.0
- netcoreapp2.1
- netcoreapp2.2
- netcoreapp3.0

Reply:
Will do!